draft-ietf-quic-applicability.md

```diff
@@ -709,6 +709,22 @@ new version uses the same process in reverse.  Servers disable validation of the
 old version, stop sending the old version in Version Negotiation packets, then
 the old version is no longer accepted.
 
+# Unreliable Datagram Service over QUIC
+
+{{?I-D.draft-ietf-quic-datagram}} specifies an QUIC extension to enable sending
+and receiving unreliable datagrams over QUIC. Other than with the use of UDP
+only, the datagram service in QUIC is congestion-controlled, and thereby
```

Comment:
What does "Other than with the use of UDP only, the datagram service in QUIC is congestion-controlled," mean?

Reply:
This probably would be cleared if it were phrased as being about unreliable datagrams, not UDP. Tunneling UDP is just one way to use datagram frames.

Reply:
I would rephrase this as "Unlike operating directly over UDP, an applications that uses the QUIC datagram service need not provide its own congestion control, per {{?RFC8085}}, as QUIC datagrams are congestion controlled."

Reply:
```suggestion
and receiving unreliable datagrams over QUIC. Unlike operating directly over UDP,
an applications that uses the QUIC datagram service need not provide its own
congestion control, per {{?RFC8085}}, as QUIC datagrams are congestion controlled.
```

Reply:
Yes, thanks. That seems clearer to me, except s/its/their